tests/ui/perfherder/graphs_view_test.jsx

```diff
@@ -226,11 +227,47 @@ test('Using select query param displays tooltip for correct datapoint', async ()
   const revision = await waitForElement(() =>
     getByText(expectedRevision.slice(0, 13)),
   );
-
+  const repoName = await waitForElement(() => getByTestId('repoName'));
   expect(graphTooltip).toBeInTheDocument();
   expect(revision).toBeInTheDocument();
+  expect(repoName).toHaveTextContent(testData[0].repository_name);
```

Comment:
We could also check for the platform name, which is `linux64` *(in our case)*.

Reply:
You don't need to use a `data-testid` here. You can use `getbyText(testData[0].repository_name)`. If it failed to find the element, that most likely means it doesn't exist. So then asserting its in the document IMO is a formality.

Reply:
We need to use test-dataid here because mozilla central appears many times in the graphViewContainer page. 
So, we need to check it for that element exactly

Reply:
Ok, that's fine then.

Reply:
Also, you should add the test for platform name as Ionut suggested.

Reply:
I do not see the second part of my suggestion though - which was to add more data to the mock file (different signature_id, so a different test).